app/src/main/java/com/example/lwilcox/lab2/SearchFragment.java

```diff
@@ -0,0 +1,127 @@
+package com.example.lwilcox.lab2;
+
+import android.content.Context;
+import android.support.v4.app.Fragment;
+import android.os.Bundle;
+import android.support.v4.app.FragmentTransaction;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.Toast;
+import java.util.ArrayList;
+
+public class SearchFragment extends Fragment {
+    public Button searchButton;
+    public  Button nextButton;
+    public Button previousButton;
+    public Button saveButton;
+    public Button viewStreamButton;
+    private View myFragmentView;
+    public EditText editText;
+    public ImageView imageView;
+    public Integer photoIndex;
+    public ArrayList<String> imageLinks;
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        myFragmentView = inflater.inflate(R.layout.fragment_search, container, false);
+
+        searchButton = (Button) myFragmentView.findViewById(R.id.search);
+        nextButton = (Button) myFragmentView.findViewById(R.id.next);
+        previousButton = (Button) myFragmentView.findViewById(R.id.previous);
+        saveButton = (Button) myFragmentView.findViewById(R.id.save);
+        viewStreamButton = (Button) myFragmentView.findViewById(R.id.viewStream);
+        editText = (EditText) myFragmentView.findViewById(R.id.editText);
+        imageView = (ImageView) myFragmentView.findViewById(R.id.imageView);
+        final FeedReaderDbHelper FeedReaderDBH = new FeedReaderDbHelper(getActivity().getApplicationContext());
```

Comment:
Activities are a subclass of context so whenever you see Context as an argument you can just pass in getActivity()